src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
Browsers cannot interpret SCSS files directly. You need to compile your SCSS file to a CSS file and link the resulting CSS file here.